docker/supabase/volumes/db/init/05-github-schema.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE IF NOT EXISTS github_users (
+  id serial PRIMARY KEY,
+  login text, -- the user name of the user, known as the `login` field in GitHub API responses
+  type text check ( type IN ('user', 'organization') )
```

Comment:
This needs to be `NOT NULL` otherwise the check is not meaningful.

Also, I think it's good to add `NOT NULL` or `DEFAULT` to where it makes sense.

Reply:
I don't think there is a safe DEFAULT in this case. I will put a NOT NULL constraint to be explicit